diff --git a/apps/website/astro.config.ts b/apps/website/astro.config.ts
index 4e868a20..c75896dc 100644
--- a/apps/website/astro.config.ts
+++ b/apps/website/astro.config.ts
@@ -30,7 +30,7 @@ if (DEV) {
 }
 
 export default defineConfig({
-  site: process.env.PUBLIC_SITE_URL || 'http://localhost:4321',
+  site: 'https://dmitrybond.tech',
   output: 'server',
   adapter: node({ mode: 'standalone' }),
   devToolbar: { enabled: false },
