diff --git a/apps/website/astro.config.ts b/apps/website/astro.config.ts
index c75896dc..0f2b4554 100644
--- a/apps/website/astro.config.ts
+++ b/apps/website/astro.config.ts
@@ -31,9 +31,14 @@ if (DEV) {
 
 export default defineConfig({
   site: 'https://dmitrybond.tech',
+  base: '/',
+  trailingSlash: 'never',
   output: 'server',
   adapter: node({ mode: 'standalone' }),
   devToolbar: { enabled: false },
+  build: {
+    assets: '_astro',
+  },
   server: { 
     port: 4321, 
     host: true,
